pom.xml

```diff
@@ -79,35 +79,35 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-    <source.version>1.7</source.version>
-    <target.version>1.7</target.version>
+    <source.version>9</source.version>
+    <target.version>9</target.version>
```

Comment:
This should still be runnable on Java 8.

Reply:
Ok, then I will set source/target version to 1.8.